src/VotesPartialDelegationUpgradeable.sol

```diff
@@ -204,7 +207,13 @@ abstract contract VotesPartialDelegationUpgradeable is
   }
 
   /**
-   * @dev Delegates votes from the sender to each `PartialDelegation._delegatee`.
+   * @notice Delegates votes from the sender to any number of `PartialDelegation._delegatee`s, up to
+   * `MAX_PARTIAL_DELEGATIONS`.
```

Comment:
if possible, highlight even more the fact that the delegations must be sorted by delegatee, contain no duplicates, and sum to less than DENOMINATOR.